extensions/kotlin/src/test/kotlin/io/opentelemetry/extension/kotlin/StrictContextWithCoroutinesTest.kt

```diff
@@ -141,20 +141,23 @@ class StrictContextWithCoroutinesTest {
         Context.current().with(ANIMAL, animal).makeCurrent().use {
             assertThat(Context.current().get(ANIMAL)).isEqualTo(animal)
             delay(10)
+            // May be on a different thread, in which case ANIMAL != animal!
```

Comment:
is this comment explaining why we're not re-asserting? If so, can you just add that bit to the comment? Otherwise, explain what it is for. :)